source/stylesheets/_navigation.scss

```diff
@@ -0,0 +1,210 @@
+
+// ==== main nav =====
+.nav-title{
+	color: $text-color-lighter;
+	font-weight: $medium-weight;
+	border-bottom: none;
+	padding: $medium-padding;
+}
+
+ul{
+	list-style-type: none;
+    @extend .center-object;
+}
+
+li {
+	float: left;
+}
+
+li a {
+    display: block;
+    text-align: center;
+    padding-left: 2.3em;
+    padding-right: 2.3em;
+    text-decoration: none;
+    border-bottom: none;
+    font-size: $subhead-size;
+}
+
+// selected navigation bar element 
+.active {
+ 	color: $emphasis_color-medium;
+ 	font-weight: $medium-weight;
+
+}
+
+.header-shadow{
+	box-shadow: 0 .1em .2em #E8E8E8;
+	width: 100%;
+	height: 5em;
+	property: float;
+	z-index: 200;
+	background-color: white;
+	position: relative;
+}
+
+.white-text{
+	color:white;
+}
+
+// moves orange button into upper nav bar on mobile
+.position-button{
```

Comment:
I'd consider making this class more specific in this case, since I bet it's really only used here? `.header-contact-button`?